README.md

```diff
@@ -114,6 +119,21 @@ Would filter your list of articles to any article with "cakephp" in the `title`
 or `content` field. You might choose to make a `get` form which posts the filter
 directly to the url, or create links manually.
 
+### Creating your GET form
+You can change the method of a form when you create it.
+
+```php
+    echo $this->Form->create(null, ['type' => 'get']);
```

Comment:
Ah ok, in the example I have running here, I'm using a get form and Prg and it's working as I expect. I'll update this then.